### ISSUE TITLE
"Map orientation" settable through "configure map" or "configure screen"

### ISSUE THREAD

**Issue by bjohas** [osmandapp/Osmand#2621]
While the the compass setting is specific to the "transport mode" view ("World/Car/..."), under "configure map" or "configure screen" there's no item for it and this it cannot be changed there. Would be good to add a menu item there for consistency.

**Comment by sonora** [comment]
Agree. I even think we could replace the extra button we currently have on the map screen by it, I have many user reports where users tapped it accidentally and do not know what exactly is going on.

Maybe we should rather replace it by a "Search button, see issue #2623. Search is a much more regularly needed basic functionality.

**Comment by bjohas** [comment]
The suggestion wasn't necessarily to replace the onscreen button, but simply to add this under "configure map/screen" too.

**Comment by vshcherb** [comment]
Agreed, make sense

**Comment by sonora** [comment]
Should be added under 'Configure map' then, I suggest below 'Map language'.
